lib/array.rb

```diff
@@ -0,0 +1,11 @@
+class Array
+
+	def my_inject(argument = 0)
```

Comment:
Indentation should be 2 spaces - in Sublime, you can set that up in your default preferences so that it'll be correct for all of your future code